task1.js

```diff
@@ -4,8 +4,24 @@ Array.prototype.myForEach = function (callback) {
     }
  }
  
+Array.prototype.myReduce = function (callback, initialValues) {
+    
+    let accamulator = ( initialValues === undefined ) ? this[0] : initialValues
+    const start = ( initialValues === undefined ) ? 1 : 0
+
+    for  (let i = start; i < this.length; i++) {
+       accamulator = callback (accamulator, this[i]) 
+    }
+    return accamulator;
+}
+
  const arr = ["a", "b", "c"]
+
+ arr.myReduce(() => {
+     console.log();
```

Comment:
Please update the usage of the method to show a more obvious use case of this method.

Reply:
Ooops, i forgot about this, yes of course i'll do some basic tests